runtime/compiler/env/j9methodServer.cpp

```diff
@@ -2245,6 +2246,13 @@ TR_ResolvedRelocatableJ9JITServerMethod::isUnresolvedMethodHandle(I_32 cpIndex)
    return true;
    }
 
+char *
```

Comment:
This method is something I removed from jitaas and should be removed from master branch as well.